pkg/reconcile/pipeline/context/impl.go

```diff
@@ -167,14 +170,36 @@ func (i *impl) AddBindingItem(item *pipeline.BindingItem) {
 }
 
 func (i *impl) BindingItems() pipeline.BindingItems {
-	return i.bindingItems
+	var allItems pipeline.BindingItems
+	for _, b := range i.bindings {
+		items, err := b.Items()
```

Comment:
Is it possible to drop the error return value here?

Reply:
not sure what do you mean - would you like that return error to the caller, instead of trying to collect as much as possible items?

Reply:
The error is captured but ignored, and also the fact that `Items()` doesn't read as a method that might return an error.

Reply:
you are right - according to the current contract `Items()` does not return error, it is just a getter.